experimental/packages/api-logs/src/ProxyLoggerProvider.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { LoggerProvider } from './types/LoggerProvider';
+import { Logger } from './types/Logger';
+import { LoggerOptions } from './types/LoggerOptions';
+import { NOOP_LOGGER_PROVIDER } from './NoopLoggerProvider';
+import { ProxyLogger } from './ProxyLogger';
+
+export class ProxyLoggerProvider implements LoggerProvider {
+  private _delegate?: LoggerProvider;
+
+  getLogger(
+    _name: string,
+    _version?: string | undefined,
+    _options?: LoggerOptions | undefined
+  ): Logger {
+    return (
+      this.getDelegateLogger(_name, _version, _options) ??
+      new ProxyLogger(this, _name, _version, _options)
+    );
+  }
```

Comment:
should these argument names start with underscore? they are used in the function.
see existing code for [`TraceProxyProvider`](https://github.com/open-telemetry/opentelemetry-js/blob/ae25ae33195f84e98f12c6d0cefca757a64950d5/api/src/trace/ProxyTracerProvider.ts#L39)

```suggestion
  getLogger(
    name: string,
    version?: string | undefined,
    options?: LoggerOptions | undefined
  ): Logger {
    return (
      this.getDelegateLogger(name, version, options) ??
      new ProxyLogger(this, name, version, options)
    );
  }
```